src/utils/maskHandler.js

```diff
@@ -26,15 +26,15 @@ export default function (fields) {
 
       case 'date':
         element.setAttribute('placeholder', '__/__/____');
-        const dateMask = IMask(element, maskDate());
+        var dateMask = IMask(element, maskDate());
```

Comment:
const

Reply:
Depois que eu coloquei o eslint com a standard eslint config ele deu esse erro: `Unexpected lexical declaration in case block.eslintno-case-declarations`